### DISCUSSION TITLE
Support fo images as doctype?

### DISCUSSION THREAD

**Discussion by horphi0815** [jonaswinkler/paperless-ng#828] [Support]
Hi,

i would like to ask whether Paperless-ng supports also images to import and archive?

BR

Phil

**Comment by jonaswinkler** [comment]
Paperless also accepts .jpg, .png, and .tiff files. These are treated as documents, and paperless performs OCR on them. Paperless is not meant to archive photos.

**Reply by horphi0815** [reply]
Thx for your fast answer.

sometimes a error occured on images. Does the content of the image is relevant for upsoad decision? 

IMG_20210118_152913.jpg
ExifOrientationError: Invalid rotation (0)

Bildschirmfoto von 2021-03-25 14-12-15.png
UnsupportedImageFormatError: 

Logo1.png

UnsupportedImageFormatError:
Logo2.png

Dateityp image/svg+xml nicht unterstützt
Logo_rgb.png

UnsupportedImageFormatError:
Logo_rgb_150.png

UnsupportedImageFormatError:
Logo_rgb_xs.png

UnsupportedImageFormatError:
Logo_sw.png

AlphaChannelError: Refusing to work on images with alpha channel
Logo_weiß.png

AlphaChannelError: Refusing to work on images with alpha channel
FLYER_1024.png

UnsupportedImageFormatError: 

Best regards,

Phil

**Comment by geomule** [comment]
> i would like to ask whether Paperless-ng supports also images to import and archive?
> 


Hi,
I'm having similar issues with the `ExifOrientationError: Invalid rotation (0)` when I try to import jpg images that I have taken with Open Notescanner app on Android.

If I do `for i in $(find . -maxdepth 1 -name "*.jpg" -print); do convert $i ${i//jpg/pdf};` done to convert the jpg to pdf, the pdf will import correctly. Its a workaround but it would be better to import the original file as I could then use Nextcloud auto-upload to dump the images automatically in the consume folder.

Any thoughts?

**Reply by geomule** [reply]
Ok, seems to be related to this https://github.com/allgood/OpenNoteScanner/issues/248

**Comment by geomule** [comment]
So here is the solution:

The problem is associate with app on the phone not assigning the correct Orientation to the exit headers. In fact, if you run:

exiftool -Orientation -n iamge.jpg

you get:

Orientation : 0

To set the orientation to 1, run:

exiftool -Orientation=1 -n iamge.jpg

This will fix the orientation problem of the jpg and Paperless will import it correctly.
